packages/config/src/layer2s/parallel.ts

```diff
@@ -0,0 +1,47 @@
+import { EthereumAddress, ProjectId } from '@l2beat/shared-pure'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { CONTRACTS, TECHNOLOGY, UPCOMING_RISK_VIEW } from './common'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('parallel')
```

Comment:
If your project is upcoming you do not have any discovery files. Discovery is a process of finding out the state of smart-contracts tied to a given application. Since you didn't deploy yet, there is nothing to discover.